documentation/modules/overview/con-key-features-kafka-connect.adoc

```diff
@@ -0,0 +1,102 @@
+// This module is included in:
+//
+// overview/assembly-key-features.adoc
+
+[id="key-features-kafka-connect_{context}"]
+= Kafka Connect concepts
+
+[role="_abstract"]
+Strimzi operates Kafka Connect in _distributed mode_, distributing data streaming tasks across worker nodes.
+A Kafka Connect cluster comprises a group of worker nodes that have the same group id, which must be unique to that cluster.
+The group id is assigned in the worker configuration through the `KafkaConnect` resource.
+The worker configuration must also specify the names of internal Kafka Connect topics that the workers will use.
+The internal topics are created by Kafka connect to store connector offset and status information.
+The names of the topics must also be unique to the Kafka Connect cluster.
+
+.Kafka Connect in distributed mode
+image:overview/kafka-concepts-kafka-connect.png[Kafka Connect interaction in distributed mode]
+
+Kafka Connect loads connector instances on start up and distributes data streaming tasks and connector configuration across the worker nodes.
+Workers run the data streaming processes for the connector instances.
+Each worker runs as a separate node to make the Kafka Connect cluster more fault tolerant.
+Work is distributed evenly through load balancing and is split into tasks.
+If there are more tasks than workers, workers are assigned multiple tasks.
+If a worker fails, its processes are automatically assigned to active workers in the Kafka Connect cluster.
+
+== Connectors
+
+Connectors copy data from external systems to Kafka or from Kafka into external systems.
+
+* Source connectors push data into Kafka.
+* Sink connectors extract data out of Kafka.
+
+Connector plugins provide the implementation for Kafka Connect to run connector instances.
+Connector instances create and orchestrate the tasks required to transfer data in and out of Kafka.
+Tasks split the work required.
+The number of tasks created relates to the number of Kafka topics and partitions.
```

Comment:
This isn't true. It's true for sink connectors that the number of tasks relates to the number of topic partitions being consumed. But for source connectors how the source data is partitioned is defined by the connector. For example a JDBC connector with `maxTasks=2` might take the set of tables to be imported, split it into two and return two task configs, one for consuming the one set of tables and the other for consuming the other.

Reply:
I've reworded this and move the content under _Tasks_

> For sink connectors, the number of tasks created relates to the number of partitions being consumed.
> For source connectors, how the source data is partitioned is defined by the connector.
> You can control the maximum number of tasks that can run in parallel by setting `tasksMax` in the connector configuration.